### PR TITLE
Enhance Data Visualization in Scenario Library with Projection Graphs

### LINKED ISSUE #347: Enhance the Data Visualization in the Scenario Library
**Is your feature request related to a problem? Please describe.**

At the moment, our Scenario Library modal shows each Scenario with a simple data visualization that includes the Recidiviz logo (see the image below).  Ideally, we would provide something more visually appealing.  The [original mocks](https://www.figma.com/file/EGJS9njuKHWdFHUS4lrHA4/%F0%9F%9A%80Recidiviz-CJStatus?node-id=910%3A130) show a data visualization that represents the projection graph of the first facility within the scenario.

Screen Shot 2020-05-08 at 12 38 36 PM

**Describe the solution you'd like**

Part of the solution is to figure out what the ultimate solution is and to decide if this is something we even want to pursues.  At the moment, here are a few options and some of their advantages and disadvantages:

1.  Leave the Recidiviz Logo visualization in place.
- Pros:
  - The work is already done
  - There aren't any performance implications
  - The case where a scenario does not yet have any facilities is already handled
- Cons:
  - Visually boring

2.  Implement the ideal visualization as defined in the mocks:
- Pros:
  - Looks good
  - Is somewhat relevant to the underlying Scenario
- Cons:
  - Will require additional database code (or updates) to allow for fetching a single facility
  - Will likely have performance implications due to additional data fetching and chart rendering
  - May require untangling some code since at the moment `useModel` (where we translate `modelInputs` into projection curves) must be used within an `EpidemicModelContext` which relies on a `LocaleDataSource`.  Coupling all of these things into this Scenario Library experience seems awkward from a code perspective.
  - Still need to handle the case where a scenario doesn't yet have any facilities

3.  Generate Random Graphs.  I have this semi-working in a local branch on my machine.
- Pros:
  - Looks good
  - Probably easier to implement than Option 2
  - The case where a scenario does not yet have any facilities is handled
- Cons:
  - Could be difficult to implement realistic looking curves
  - Might have to be smart about the curve generation such that generated curves are static for a given scenario.  Otherwise it might be confusing if the curves are constantly changing for the same scenario.

### PR DESCRIPTION
**Pull request by neeharmb**
## Description of the change

> The Scenario Library now displays the projection graph and Rt value for the first facility in a scenario instead of the Recidiviz logo. 

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #347

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing has been performed locally

Basic testing: 
Screen Shot 2020-06-24 at 4 30 53 PM

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/page-multi-facility/ScenarioLibraryModal.tsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import useCurrentUserId from "../hooks/useCurrentUserId";
 import useRejectionToast from "../hooks/useRejectionToast";
```
Comment (jessex) [inline src/page-multi-facility/ScenarioLibraryModal.tsx:15]: Another lint check on L8 (I can't directly comment on it) to take care of.

File: src/page-multi-facility/FacilityChart.tsx [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+`;
+
+interface FacilityChartProps {
+  facility: Facility | undefined;
+  facilityRtData: RtValue | undefined;
+}
+
+const FacilityChart: React.FC<{
+  scenarioId: string;
+  indexOfFacility: number;
```
Comment (macfarlandian) [inline src/page-multi-facility/FacilityChart.tsx:38]: i'm not so sure there's much benefit to having this passed as a prop? if anything we might want to change the index we use _inside_ the component once the data is loaded (e.g. choose the first facility that has valid data, or the one that is most recently updated - not that I'm necessarily suggesting you do that here, just that it's probably a more likely future use case than arbitrarily choosing a different index from outside this component)

File: src/page-multi-facility/FacilityChart.tsx [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+          setFacility(facility);
+          setFacilityRtData(facilityRtData);
+        }
+      }
+    }
+    fetchFacility();
+    return () => {
+      mounted = false;
+    };
+  }, []);
```
[the remark this answers is not in the document]
Comment (macfarlandian) [reply src/page-multi-facility/FacilityChart.tsx:61]: to me it seems sensible to include both of these dependencies; if either of them changed you would need new data to reflect those changes anyway

File: src/page-multi-facility/FacilityChart.tsx [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+  const [facilityRtData, setFacilityRtData] = useState<RtValue | undefined>();
+  const { data: localeDataSource } = useLocaleDataState();
+
+  useEffect(() => {
+    let mounted = true;
+    async function fetchFacility() {
+      const facilities = await getFacilities(scenarioId);
+      if (facilities) {
+        const facility = facilities[indexOfFacility];
+        const facilityRtData = await getRtDataForFacility(facility);
```
Comment (macfarlandian) [inline src/page-multi-facility/FacilityChart.tsx:50]: I would recommend not "shadowing" the state variable names like this; it doesn't break anything per se but it makes things harder to read

**Pull request merged**